README.md

```diff
@@ -1 +1,76 @@
-# hello-microverse-TSE-practise
\ No newline at end of file
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Project Name
+
+> One paragraph statement about the project.
```

Comment:
- [ ] It will be nice to add a `project title` and a `brief description` to the readme. This will ensure our project has a descriptive readme.